frontend/src/app/testResults/mocks/resultsByAllFacilities.mock.tsx

```diff
@@ -1,4 +1,4 @@
-const TEST_RESULTS_BY_ALL_FACILITIES = [
+const data = [
```

Comment:
I pulled out the actual mock data from the part that formats the shape of the returned object to make things easier to read. All the files here are doing the same thing.